From adcbf284a730d3421d741389f114e6228ef489fb Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Wed, 22 Dec 2010 10:01:38 -0500
Subject: [PATCH 2/3] testfail must fail -- so lets state that ;)

---
 test/CMakeLists.txt |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index f6d66b9..e35734a 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -37,6 +37,8 @@ target_link_libraries( testRWConsistency ${Boost_LIBRARIES} GDF )
 
 add_test( NAME testok COMMAND testok )
 add_test( NAME testfail COMMAND testfail )
+set_tests_properties( testfail PROPERTIES WILL_FAIL TRUE )
+
 add_test( NAME testCreateGDF COMMAND testCreateGDF )
 add_test( NAME testRWConsistency COMMAND testRWConsistency )
 
-- 
1.7.2.3

